cmd/operator-sdk/build/cmd.go

```diff
@@ -141,6 +143,27 @@ func verifyTestManifest(image string) error {
 	return nil
 }
 
+func createBuildCommand(imageBuilder, context, dockerFile, image, imageBuildArgs string) (*exec.Cmd, error) {
```

Comment:
`imageBuildArgs ...string` so we can pass any number of string args without having to create a new string from them. Then you can loop over `imageBuildArgs` like so:
```go
for _, bargs := range imageBuildArgs {
	if bargs != "" {
		splitArgs := strings.Fields(bargs)
		args = append(args, splitArgs...)
	}
}
```